src/main/resources/test_files/runtime/ControlIterators/TypeswitchExpression1.jq

```diff
@@ -0,0 +1,9 @@
+(:JIQS: ShouldRun; Output="string" :)
```

Comment:
Is there a reason why there are only ItemType tests (i.e., SequenceTypes with an exact cardinality of one)?

Reply:
No there was not real reason for that. I just did not realize. 
I added the arity now.
Thank you